pyensembl/species.py

```diff
@@ -145,7 +145,7 @@ def normalize_species_name(name):
 def find_species_by_name(species_name):
     latin_name = normalize_species_name(species_name)
     if latin_name not in Species._latin_names_to_species:
-        raise ValueError("Species not found: %s" % species_name)
+        raise ValueError("Species not found: %s; For non-Ensembl Data, see https://github.com/openvax/pyensembl#non-ensembl-data" % species_name)
```

Comment:
Small nit, can you change this to:

```
raise ValueError("Species not found: %s; for non-Ensembl data see https://github.com/openvax/pyensembl#non-ensembl-data" % (species_name,))
```

Also, if you increase the library version to 2.2.7 then it will auto-deploy to PyPI on merge